include/nil/blueprint/asserts.hpp

```diff
@@ -62,6 +66,24 @@ namespace nil {
                 abort_process();
             }
         }
+
+        [[noreturn]] void todo(const char *msg, const char *filename, unsigned line) {
+            std::cerr << "NOT YET IMPLEMENTED at " << filename << ":" << line << std::endl;
+            std::cerr << '\t' << msg << std::endl;
+            abort_process();
+        }
+
+        [[noreturn]] void todo_with_link(const char *msg, const char *link, const char *filename, unsigned line) {
+            char *new_msg;
+            std::strcpy(new_msg, msg);
```

Comment:
`std::strcpy` [requires](https://en.cppreference.com/w/cpp/string/byte/strcpy) a pointer with allocated memory (amount must be large enough) instead of uninitialized `new_msg`. Similar for `std::strcat`. I would suggest to use `std::string` instead as a temporary value.

Reply:
Oh sure, you're right. I don't know, why I even looked at these C-functions.